src/posts/2020-10-14-wcag2-documents.md

```diff
@@ -0,0 +1,111 @@
+---
+layout: post
+title: Introduction to WCAG 2.0
```

Comment:
```suggestion
title: Introduction to WCAG 2.x
```

We'll want to use "2.x" instead of "2.0" to indicate that this post is about versions, 2.0, 2.1, and the current version 2.2, as well as any potential future releases before version 3.0.